src/main/java/com/GeorgesServer/app/com/GeorgesServer/response/IResponseBuilder.java

```diff
@@ -0,0 +1,7 @@
+package com.GeorgesServer.app.com.GeorgesServer.response;
+
+public interface IResponseBuilder {
```

Comment:
This will eventually contain methods like setBody, setHeaders and so forth, but right now it satisfies simpleGet, Post, Put